src/main/webapp/resources/js/apis/projects/settings.js

```diff
@@ -0,0 +1,60 @@
+import axios from "axios";
+import { setBaseUrl } from "../../utilities/url-utilities";
+
+export const fetchProcessingInformation = async (projectId) =>
+  fetch(setBaseUrl(`ajax/projects/${projectId}/settings/priorities`))
```

Comment:
Just a question. Is there a specific reason you are using the fetch API here and in the two functions below it?

Reply:
Switch to axios in d4d2e73, but fetch is a good standard now so it is safe to use.